packages/vscode-extension/PRERELEASE.md

```diff
@@ -72,6 +72,6 @@ Pre-release is meant for those eager to try the latest Teams Toolkit features an
 
 ### What about my existing Teams Toolkit projects
 
-The changes in this pre-release require upgrades to the TeamsFx configuration files. We recommend that you create a new app using this version. In the future, we'll provide a way to automatically upgrade existing Teams apps that were created with a previous version of Teams Toolkit.
+The changes in this pre-release require upgrades to the TeamsFx configuration files. We recommend that you create a new app using this version. We have also provided a way to automatically upgrade existing Teams apps that were created with a previous version of Teams Toolkit.
```

Comment:
Why do we recommend creating a _new_ app if we have automatic upgrades? Should we instead not that if you open an old project it will be upgraded? Does this make sense?:

"The changes in this pre-release require an upgrade to the Teams Toolkit configuration files. When you open a project that was created with a previous version, Teams Toolkit will automatically upgrade the project files and create a backup of the older versions in a /.backup directory."

Reply:
Updated.